<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabRunByRun</class>
 <widget class="QWidget" name="TabRunByRun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Detectors</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>CB</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>315</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="TQtWidget" name="widget" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QWidget" name="widget_3" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QPushButton" name="pushButton">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/view_refresh.png</normaloff>:/data/view_refresh.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonZoomIn">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/forward.png</normaloff>:/data/forward.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonZoomOut">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/arrow_down.png</normaloff>:/data/arrow_down.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_4" native="true"/>
              </item>
              <item>
               <widget class="QWidget" name="widget_5" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Taps 1</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>315</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="TQtWidget" name="widgetTaps1" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_7" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QWidget" name="widget_8" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QPushButton" name="buttonTaps1R">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/view_refresh.png</normaloff>:/data/view_refresh.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonTaps1Plus">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/forward.png</normaloff>:/data/forward.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonTaps1Minus">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/arrow_down.png</normaloff>:/data/arrow_down.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_9" native="true"/>
              </item>
              <item>
               <widget class="QWidget" name="widget_10" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Taps 2</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QScrollArea" name="scrollArea_3">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>315</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="TQtWidget" name="widgetTaps2" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_11" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QWidget" name="widget_12" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QPushButton" name="buttonTaps2R">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/view_refresh.png</normaloff>:/data/view_refresh.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonTaps2Plus">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/forward.png</normaloff>:/data/forward.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonTaps2Minus">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/arrow_down.png</normaloff>:/data/arrow_down.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_13" native="true"/>
              </item>
              <item>
               <widget class="QWidget" name="widget_14" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Taps 3</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QScrollArea" name="scrollArea_4">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_4">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>315</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="TQtWidget" name="widgetTaps3" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_15" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QWidget" name="widget_16" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <widget class="QPushButton" name="buttonTaps3R">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/view_refresh.png</normaloff>:/data/view_refresh.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonTaps3Plus">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/forward.png</normaloff>:/data/forward.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonTaps3Minus">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="Resources.qrc">
                     <normaloff>:/data/arrow_down.png</normaloff>:/data/arrow_down.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_17" native="true"/>
              </item>
              <item>
               <widget class="QWidget" name="widget_18" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Physics Analysis</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QScrollArea" name="scrollArea_5">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>522</width>
            <height>366</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="TQtWidget" name="widgetPA" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_19" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QWidget" name="widget_20" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QPushButton" name="buttonPAR">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="Resources.qrc">
                 <normaloff>:/data/view_refresh.png</normaloff>:/data/view_refresh.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonPAPlus">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="Resources.qrc">
                 <normaloff>:/data/forward.png</normaloff>:/data/forward.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonTapsPAM">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="Resources.qrc">
                 <normaloff>:/data/arrow_down.png</normaloff>:/data/arrow_down.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_21" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="widget_22" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TQtWidget</class>
   <extends>QWidget</extends>
   <header>TQtWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
